docs/src/pages/wrap.mdx

```diff
@@ -0,0 +1,30 @@
+import PropertyTable from '../components/propertyTable';
+import wrapSchema from '../../../api/components/wrap.schema.json';
+
+# Wrap
+
+This component can be used to display a long list of components, if they get out of the screen the wrap component will make the overflowing ones use a new row.
+
+## Examples
+
+### Simple wrap
+
```

Comment:
```suggestion
`wrap` component work like `flex` components, but instead of trying to fit children in the available space, it will allow you to set fixed width/height and make it display in multiple rows if it overflow the parent.

```